test/clj/rems/api/test_applications.clj

```diff
@@ -562,6 +577,39 @@
         (is (response-is-not-found?
              (api-response :get (str "/api/applications/" app-id) nil
                            api-key applicant)))))
+    (testing "can delete application with attachments"
+      (let [form-id (test-helpers/create-form! {:form/fields [{:field/id "att"
+                                                               :field/type :attachment
+                                                               :field/title {:en "x" :fi "x" :sv "x"}
+                                                               :field/optional false}]})
+            cat-id (test-helpers/create-catalogue-item! {:form-id form-id})
+            app-id (test-helpers/create-application! {:catalogue-item-ids [cat-id]
+                                                      :actor applicant})
+            att-id (-> (request :post (str "/api/applications/add-attachment?application-id=" app-id))
+                       (assoc :params {"file" filecontent})
+                       (assoc :multipart-params {"file" filecontent})
```

Comment:
Do you know why we need both params and mutipart params?

Reply:
No, I copypasted them from the attachment tests :( I'll investigate a bit.

Reply:
Yeah, they can just be dropped it seems :)